<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>897</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>软包装检测</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/新前缀/165.png</normaloff>:/新前缀/165.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	
	background-color: rgb(207, 224, 247);
	
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>100</width>
    <height>50</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(207, 224, 247), stop:1 rgba(252, 252, 252, 255));

	
	
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>4</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>302</width>
          <height>10000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>百度综艺简体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabWidget{	
font: 11pt &quot;百度综艺简体&quot;;

	background-color: rgb(190, 43, 46);
border-radius: 1px;
border:1px groove gray;
border-style: outset;
	color: rgb(253, 253, 253);

}</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_zc">
         <attribute name="title">
          <string>正常</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="zc">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>10000</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QListView{
 font: 11pt &quot;微软雅黑&quot;;
 background-color: rgb(252, 252, 252);
border-radius: 5px;
border:2px groove gray;
border-style: outset;
	color: rgb(0, 0, 0);
}</string>
            </property>
            <property name="defaultDropAction">
             <enum>Qt::MoveAction</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_yc">
         <attribute name="title">
          <string>异常</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="yc">
            <property name="styleSheet">
             <string notr="true">QListView{
 font: 11pt &quot;微软雅黑&quot;;
 background-color: rgb(252, 252, 252);
border-radius: 5px;
border:2px groove gray;
border-style: outset;
	color: rgb(0, 0, 0);
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
 font: 11pt &quot;微软雅黑&quot;;
 background-color: rgb(252, 252, 252);
border-radius: 5px;
border:2px groove gray;
border-style: outset;
color:rgb(163, 202, 235)
}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">QPushButton

{
	font: 10pt &quot;百度综艺简体&quot;;
	
	background-color: rgb(66, 93, 138);
border-radius:10px;
border:2px groove gray;
border-style: outset;
	color: rgb(252, 252, 252);
}
</string>
        </property>
        <property name="text">
         <string>点击生成热点图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>390</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
 font: 11pt &quot;微软雅黑&quot;;
 background-color: rgb(252, 252, 252);
border-radius: 5px;
border:2px groove gray;
border-style: outset;
color:rgb(163, 202, 235)
}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>26</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar{
	background-color: rgb(207, 224, 247);
	
	color: rgb(45, 96, 151);
}
</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="styleSheet">
     <string notr="true">QMenu{
		
	font: 10pt &quot;百度综艺简体&quot;;
	
	background-color: rgb(66, 93, 138);
border-radius:3px;
border:2px groove gray;
border-style: outset;
	color: rgb(252, 252, 252);
}</string>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="upload"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="font">
     <font>
      <family>百度综艺简体</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu{
	font: 10pt &quot;百度综艺简体&quot;;
	
	background-color: rgb(66, 93, 138);
border-radius:3px;
border:2px groove gray;
border-style: outset;
	color: rgb(252, 252, 252);
}</string>
    </property>
    <property name="title">
     <string>测试</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="font">
     <font>
      <family>百度综艺简体</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu{
font: 10pt &quot;百度综艺简体&quot;;
	
	background-color: rgb(66, 93, 138);
border-radius:3px;
border:2px groove gray;
border-style: outset;
	color: rgb(252, 252, 252);
}</string>
    </property>
    <property name="title">
     <string>运行</string>
    </property>
    <addaction name="action1"/>
    <addaction name="action1_2"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="font">
     <font>
      <family>百度综艺简体</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu{
	font: 10pt &quot;百度综艺简体&quot;;
	
	background-color: rgb(66, 93, 138);
border-radius:3px;
border:2px groove gray;
border-style: outset;
	color: rgb(252, 252, 252);
}</string>
    </property>
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actiontui"/>
    <addaction name="actionban"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="font">
     <font>
      <family>百度综艺简体</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>6</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu{
	font: 10pt &quot;百度综艺简体&quot;;
	
	background-color: rgb(66, 93, 138);
border-radius:3px;
border:2px groove gray;
border-style: outset;
	color: rgb(252, 252, 252);
}</string>
    </property>
    <property name="title">
     <string>分析</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar{
		background-color: rgb(207, 224, 247);
	
}</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar{
	background-color: rgb(207, 224, 247);
	}</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar{
	background-color: rgb(207, 224, 247);
	
}</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="upload"/>
   <addaction name="action1"/>
   <addaction name="action1_2"/>
   <addaction name="fx"/>
   <addaction name="actiontui"/>
  </widget>
  <action name="upload">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/新前缀/线性上传文件图标.png</normaloff>:/新前缀/线性上传文件图标.png</iconset>
   </property>
   <property name="text">
    <string>上传文件</string>
   </property>
   <property name="iconText">
    <string>上传文件</string>
   </property>
   <property name="font">
    <font>
     <family>百度综艺简体</family>
    </font>
   </property>
  </action>
  <action name="action1">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/新前缀/面性播放.png</normaloff>:/新前缀/面性播放.png</iconset>
   </property>
   <property name="text">
    <string>开始检测</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/新前缀/未标题-1.png</normaloff>:/新前缀/未标题-1.png</iconset>
   </property>
   <property name="text">
    <string>中止检测</string>
   </property>
  </action>
  <action name="actiontui">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/新前缀/线性搜索图标.png</normaloff>:/新前缀/线性搜索图标.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionban">
   <property name="text">
    <string>版本信息</string>
   </property>
  </action>
  <action name="fx">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/新前缀/线性电子技术图标.png</normaloff>:/新前缀/线性电子技术图标.png</iconset>
   </property>
   <property name="text">
    <string>数据分析</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
